pennylane/wires.py

```diff
@@ -695,5 +696,7 @@ def __rxor__(self, other):
         return Wires((set(_process(other)) ^ set(self.labels)))
 
 
+WiresLike = Union[Wires, Iterable[Hashable]]
```

Comment:
```suggestion
WiresLike = Union[Wires, Iterable[Hashable], Hashable]
```
For example, we will accept `wires=0`.  0 is an individual wire label, not an iterable of wire labels.